### DISCUSSION TITLE
EssX Github Wiki displayes wrong release version and links to old EssX page with Error

### DISCUSSION THREAD

**Discussion by Chris6ix** [EssentialsX/Essentials#4925] [General]
Hey, just wanted to let you guys know that the small "banner" on the bottom of the EssX Wiki pages displays version `2.17.2` as the latest release, although it should be `2.19.4`
![image](https://user-images.githubusercontent.com/65610536/171843681-c540d3d4-ed22-4a3c-8033-b1c14aa8d35c.png)

Edit: also noticed that clicking on the "build" brings you to an old EssX page...so the build number is probably also wrong
The redirect on top of the old EssX page is not the same as the real EssX page and also displays a security warning...
![image](https://user-images.githubusercontent.com/65610536/171844838-47b69fcd-f5f5-4588-911a-98e97d566e1a.png)

![image](https://user-images.githubusercontent.com/65610536/171844793-97b93320-3101-4604-9208-33dbe28101b6.png)
It redirects to `https://essentialsx.cf/` but the EssX page is `https://essentialsx.net/`

**Comment by osfanbuff63** [comment]
Which page did you take the screenshot on? I'm unable to reproduce. This screenshot was taken on the [wiki's index page](https://essentialsx.net/wiki)
![Screenshot 2022-06-03 072404](https://user-images.githubusercontent.com/91388253/171844872-e6e4d378-0c07-4e53-a3ff-e970ebe8393b.png)

Also, in the future, you can report bugs like this in the [EssentialsX/Wiki repo](https://github.com/EssentialsX/Wiki). On the build front...that page should probably just link to the EssX download dev build site, as it's been a long time since I've heard someone direct someone to Jenkins for dev builds.

Edit: Most likely fake/malicious site, don't go to it. You might be on one to start with, but idk.

**Reply by Chris6ix** [reply]
I took that screenshot here on Github (https://github.com/EssentialsX/Essentials/wiki) ...so it looks like the EssX Github Wiki is very out of date

**Reply by osfanbuff63** [reply]
That's possible, given I don't think we use it. Either way, we should probably fix that redirect, given the page it is taken to is essentialsx.github.io (but someone else might have put GitHub Pages on that, unless we verified it, but I'm not sure that's possible)

Edit: Appears like it's that way in the EssentialsX/essentials.github.io repo, so I'll change that.

**Reply by Chris6ix** [reply]
Yeah...if the GitHub Wiki itself isn't used (and updated) anymore, then it should probably be completely emptied with just a link to the new one (https://essentialsx.net/wiki/Home.html)
I also updated the name of this discussion to make clear that it's the GitHub Wiki that is meant

**Reply by osfanbuff63** [reply]
EssentialsX/essentialsx.github.io#3 (which was just merged) fixed the wrong redirect link on that site. The wiki might still need to be cleared though.

**Reply by mdcfe** [reply]
The wiki at https://github.com/EssentialsX/Essentials/wiki is no longer used. It is still available in the short term as an archive of the content with working links between pages. It will be removed at some point in the near future, but most likely not until the website rewrite has been completed (which itself is on hold due to a lack of time).
